src/traffic_server/traffic_server.cc

```diff
@@ -77,7 +77,7 @@ extern "C" int plock(int);
 #include "tscore/I_Layout.h"
 #include "I_Machine.h"
 #include "records/I_RecordsConfig.h"
-#include "records/I_RecProcess.h"
+#include "RecProcess.h"
```

Comment:
Since ` RecProcess.h` is an header file used by other subsystems, shouldn't it be in the` include/` directory?

Reply:
Maybe, but there isn't an include/iocore directory and everything that depends on this already has iocore/eventsystem in their include path.